config/webhook/manifests.yaml

```diff
@@ -23,15 +23,15 @@ webhooks:
     - CREATE
     - UPDATE
     resources:
-    - baremetalsets
+    - controlplanes
 - clientConfig:
     caBundle: Cg==
     service:
       name: webhook-service
       namespace: system
-      path: /validate-osp-director-openstack-org-v1beta1-controlplane
+      path: /validate-osp-director-openstack-org-v1beta1-baremetalset
   failurePolicy: Fail
-  name: vcontrolplane.kb.io
+  name: vbaremetalset.kb.io
```

Comment:
Might need to change this to the new name

Reply:
ack, this file is generate so what I missed is the changing the annotation above. Will fix...

Reply:
@abays fixed in latest commit